README.md

```diff
@@ -89,6 +89,12 @@ source ~/.virtualenvs/cb/bin/activate
 ```
 pip install -r requirements.txt
 ```
+Troubleshoot problem building `psycopg2`:
```

Comment:
I'd put this in the "Install Mac OS dependencies > Troubleshooting" category above, since this is a mac-specific problem

Reply:
Will do.